src/plugins/query_enhancements/server/utils/plugins.ts

```diff
@@ -60,9 +60,10 @@ export const OpenSearchEnhancements = (client: any, config: any, components: any
     needBody: true,
   });
   enhancements.promqlQuery = createAction(client, components, {
-    endpoint: URI.PROMQL,
+    endpoint: `${URI.DIRECT_QUERY_SYNC}`,
```

Comment:
nit
```suggestion
    endpoint: URI.DIRECT_QUERY_SYNC,
```

also this is used by `core.opensearch.legacy.createClient` to create the legacy client. I heard that legacy clients are deprecating, it might be better to directly use `client.transport.request` to query, like here
https://github.com/opensearch-project/OpenSearch-Dashboards/blob/b93d6fad6843040413ac4b7ce1c43ed299867352/src/plugins/query_enhancements/server/connections/clients/sql_connection_client.ts#L26-L30